src/register/control.rs

```diff
@@ -134,3 +180,30 @@ pub fn read() -> Control {
         () => unimplemented!(),
     }
 }
+
+/// Writes to the CPU register.
+#[inline]
+pub unsafe fn write(_control: Control) {
```

Comment:
The use of `_control` as a name is a bit unconventional, because usually leading underscores indicate unused parameters. It's fine to call it `control` here and then shadow it below with a `let control = control.bits();`.

Reply:
Other register `write` functions do the same, since the value isn't used when the `#[cfg(not(cortex_m))]` path is active